projects/plugins/social/CHANGELOG.md

```diff
@@ -5,6 +5,19 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 1.5.0-beta - 2022-10-27
+### Added
+- Display broken connections to user in editor [#25803]
+- Integrate the ConnectionError react component to the Social plugin. [#26904]
+- Reshare: Added the reshare UI to the block editor extension [#25993]
+
+### Changed
+- Updated package dependencies. [#25993, #26640, #26683, #26705, #26716, #26790, #26791, #26808, #26826, #26829, #26851, #27089]
+
+### Fixed
+- Social: Fix the connection test endpoint URL [#26892]
+- Social: Fix the path to the connections URL in the editor [#26932]
+
```

Comment:
Changes that were made on 1.4.2 release branch were never ported back to the trunk so this part of the changelog is missing.
```suggestion

## 1.4.2 - 2022-10-20
### Fixed
- Social: Fix the path to the connections URL in the editor [#26932]

## 1.4.1 - 2022-10-19
### Fixed
- Social: Fix the connection test endpoint URL [#26892]
```

Reply:
Oh no! I'll add that back in. Do you think there could be anything else missing?